src/app/components/home/home.component.ts

```diff
@@ -1,84 +1,35 @@
 import { Component, OnInit, ViewEncapsulation } from "@angular/core";
+import { CMS } from "@baw-api/cms/cms.service";
 import { ProjectsService } from "@baw-api/project/projects.service";
 import { SecurityService } from "@baw-api/security/security.service";
 import { projectsMenuItem } from "@components/projects/projects.menus";
 import { PageComponent } from "@helpers/page/pageComponent";
 import { Project } from "@models/Project";
-import { AppConfigService } from "@services/app-config/app-config.service";
 import { Card } from "@shared/cards/cards.component";
 import { List } from "immutable";
 import { map, mergeMap, takeUntil } from "rxjs/operators";
 import { homeCategory, homeMenuItem } from "./home.menus";
 
 @Component({
   selector: "baw-home",
-  styles: [
-    `
-      #website-title {
-        color: white;
-        font-weight: bold;
-        font-size: 10rem;
-        font-family: "Arial";
-        filter: drop-shadow(2px 2px 4px rgba(0, 0, 0, 0.7));
-        padding-bottom: 15rem;
-      }
-
-      section {
-        background-color: #fff;
-        filter: drop-shadow(2px 2px 4px rgba(0, 0, 0, 0.7));
-      }
-    `,
-  ],
-  template: `
-    <div id="hero-image">
-      <baw-cms [page]="page"></baw-cms>
-      <div class="container-fluid" style="padding-bottom: 3rem;">
-        <section class="container">
-          <h2>Some of the projects you can access</h2>
-          <p>
-            You can browse some public projects and audio recordings without
-            logging in. To participate in the analysis work you will need to Log
-            in with an existing account or Register for a new account.
-          </p>
-          <baw-cards [cards]="projectList">
-            <ng-container>
-              <button
-                class="m-auto btn btn-outline-primary"
-                [routerLink]="projectsLink"
-              >
-                More Projects
-              </button>
-            </ng-container>
-          </baw-cards>
-        </section>
-
-        <div class="float-right">
-          <small class="alert alert-secondary m-0">
-            Photo Credit: QUT Ecoacoustic Staff
-          </small>
-        </div>
-      </div>
-    </div>
-  `,
+  styleUrls: ["./home.component.scss"],
+  templateUrl: "./home.component.html",
   // tslint:disable-next-line: use-component-view-encapsulation
   encapsulation: ViewEncapsulation.None,
```

Comment:
This shouldn't be needed anymore

Reply:
I can't even remember what it did?

Reply:
It was applying styles to the cms page. It was a dodgy solution that's no longer needed